pkg/cmd/roachtest/test_runner.go

```diff
@@ -926,6 +933,34 @@ func (r *testRunner) runWorker(
 	}
 }
 
+// destroyCluster runs cluster destroy in a goroutine and adds 1 to the wait group if the wsit is false
+// if the wait is true, it waits till the cluster is destroyed
+// if the cluster is nil, it just returns
+func (r *testRunner) destroyCluster(
+	ctx context.Context,
+	clusterDestroyWg *sync.WaitGroup,
+	c *clusterImpl,
+	l *logger.Logger,
+	wait bool,
+) {
+	if c == nil {
+		// all good - nothing to destroy
+		return
+	}
+	if c.IsLocal() || wait {
+		// local cluster - do not use goroutine as we generally recreate the cluster.
```

Comment:
I don't quite follow this comment. Is it because you can only have one local cluster so it must be done sequentially?

Reply:
Yes, without this condition CI fails as it uses local cluster.

Reply:
Let's state that explicitly in the comment; instead of the above, something like this,

> N.B. multiple local clusters aren't supported, hence we must use a blocking call.